### DISCUSSION TITLE
Manual tagging

### DISCUSSION THREAD

**Discussion by ua741** [ente-io/ente#955] [Enhancements]
- Support adding tags
- Search by tags

**Comment by threat** [comment]
Please keep [this comment](https://github.com/ente-io/photos-app/issues/1473#issuecomment-1793807567) in mind when implementing.

**Comment by thz** [comment]
This is related, right? https://github.com/ente-io/ente/discussions/543

**Comment by ezzra** [comment]
Iam just testing ente first time, is there actually no way yet to add tags manually? it looks for example that in public collab links, the name of the uploader is added as a tag. But I would need to add and manage tags on my own, ente seems to have lots of great features but not such a basic one?

**Reply by zilexa** [reply]
There is no such thing as tagging at all in Ente. I just discovered this :(

**Comment by ua741** [comment]
Tracking this request here: https://github.com/ente/ente/discussions/543
